.travis.yml

```diff
@@ -26,6 +20,10 @@ env:
     - CFLAGS="-Werror -Wall -Wextra -Wno-error=sign-compare -Wno-error=unused-parameter -Wno-error=missing-field-initializers"
 
 before_install:
+  - |
+    if [ "$TRAVIS_OS_NAME" == linux ]; then
+      sudo apt-get install -y libgudev-1.0-dev libxml++2.6-dev
```

Comment:
Any particular reason for using the `-y` option here (and in the FreeBSD block)? I'd like to keep the number of pulled in packages to a minimum just to be sure we don't introduce any accidental build dependencies.

Reply:
To avoid the job timing out on user input. `apt` addon also uses `-y`.

Reply:
Also, I'm not very familar with `apt-get`. Maybe it assumes `-y` if run non-interactively but the manpage doesn't explicitly say so.

Reply:
I've added `-q` and `--no-install-suggests --no-install-recommends` as `apt` addon did.